freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientBuilder.java

```diff
@@ -169,7 +166,7 @@ public FreeIpaClient build(boolean withPing) throws URISyntaxException, IOExcept
                 additionalHeaders.forEach(request::addHeader);
                 additionalHeadersStickySessionFirstRpc.forEach(request::addHeader);
                 try (CloseableHttpResponse response = client.execute(request)) {
-                    if (UNAVIALLBE_PING_HTTP_RESPONSES.contains(response.getStatusLine().getStatusCode())) {
+                    if (isUnreachableHttpStatus(response.getStatusLine().getStatusCode())) {
```

Comment:
It is strange that the catch clause after this is LOGGER.debug. Might be LOGGER.error()

catch (Exception e) {
                String msg = "Ping failed";
                LOGGER.debug(msg, e);

Reply:
I will break this part out into a separate PR.